orion/pkg/controllers/achieve_controller_test.go

```diff
@@ -0,0 +1,215 @@
+package controllers_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"github.com/ahsu1230/mathnavigatorSite/orion/pkg/domains"
+	"github.com/ahsu1230/mathnavigatorSite/orion/pkg/services"
+	"github.com/stretchr/testify/assert"
+	"io"
+	"net/http"
+	"testing"
+)
+
+//
+// Test Get All
+//
+func TestGetAllAchievements_Success(t *testing.T) {
+	mas.mockGetAll = func() ([]domains.Achieve, error) {
+		return []domains.Achieve{
+			{
+				Id:      1,
+				Year:    2020,
+				Message: "message1",
+			},
+			{
+				Id:      2,
+				Year:    2021,
+				Message: "message2",
+			},
+		}, nil
+	}
+	services.AchieveService = &mas
+
+	// Create new HTTP request to endpoint
+	recorder := sendHttpRequest(t, http.MethodGet, "/api/achievements/v1/all", nil)
```

Comment:
Make sure to update `handler.go` in the routers package to make sure it will correctly take in this API endpoint